### PR TITLE
Add https://github.com/rmassei/ymia_python_event_series_material

### PR DESCRIPTION
**Pull request by SeverusYixin**
closes #236

### CODE REVIEW

**Comment by SeverusYixin** [comment]
surprise @haesleinhuepf

File: resources/nfdi4bioimage.yml [hunk 1]
```diff
@@ -4438,4 +4438,14 @@ resources:
...
   - https://doi.org/10.5281/zenodo.13908108
+  
+- authors: rmassei, haesleinhuepf
+  description: This repository offer access to teaching material and useful resources
+    for the YMIA - Python-Based Event Series.
+  last_updated: '2024-10-10T10:15:50+00:00'
+  name: ymia_python_event_series_material
+  publication_date: '2024-10-09T12:22:31+00:00'
+  tags: No tags available
+  type: MIT License
```
Comment (haesleinhuepf) [inline resources/nfdi4bioimage.yml:4449]: Shouldn't this be "license" and not "type"? @SeverusYixin

Comment (SeverusYixin) [reply resources/nfdi4bioimage.yml:4449]: Yes, I'm still working on it, this PR just means that we can take the info from another git repository and set up a PR on our git!

**Comment by haesleinhuepf** [comment]
Can this be closed @SeverusYixin ?

**Pull request closed without merging**